### ISSUE TITLE
[A11yInspect] Fail: 2.4.3 A in /home/runner/work/test-app/test-app/src/sample.jsx

### ISSUE THREAD

**Issue by sitaram-r** [sitaram-r/test-app#13]
## ❌ Accessibility Fail

**Rule:** 2.4.3 A
**File:** `/home/runner/work/test-app/test-app/src/sample.jsx`
**Line:** 1, **Column:** 1

### Description
Incorrect nesting of interactive HTML elements (Major)

### Context
```html
<div class="modal">
          <p>Are you sure?</p>
          <button>Yes</button>
        </div>
```

### Selector
`html > body > div > div > video > audio > div:nth-of-type(6)`

---
*Generated by [A11yInspect](https://github.com/barrierbreak/vscode-scally)*
